src/components/Navbar/Account.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+
+interface NavbarAccount {
+  isUserSignedIn?: boolean
+}
+
+const NavbarAccount = ({ isUserSignedIn }: NavbarAccount) => {
+  const router = useRouter()
+
+  const handleSignInClick = () => {
+    // TODO: Open SignIn modal
+  }
+
+  const handleSignUpClick = () => {
+    router.push('/sign-up')
+  }
+
+  const handleAvatarClick = () => {
+    // TODO: Expand dropdown or sth
+  }
+
+  const renderAvatarButton = () => (
+    <button className="navbar__button-avatar" onClick={handleAvatarClick}>
```

Comment:
This css class is not working. You mean this?
```suggestion
    <button className="navbar__button navbar__button--avatar" onClick={handleAvatarClick}>
```

Reply:
Thanks 😅